main.py

```diff
@@ -35,6 +42,23 @@ def setup(self):
 
         So, if you have any setup routine, insert it here.
         """
+        # Validate Spec
+        self.napps_manager = NAppsManager(self.controller)
+        enabled_path = str(self.napps_manager._enabled_path)
+        if enabled_path[:2] == "//":
+            enabled_path = enabled_path[1:]
+        yml_file = enabled_path + "/.installed/kytos/mef_eline/openapi.yml"
+        spec_dict, spec_url = read_from_filename(yml_file)
+        try:
+            spec_error = validate_spec(spec_dict)
+        except ValueError as err:
+            raise BadRequest(err) from err
```

Comment:
Since it is the Napp startup, I'm not sure we should raise a BadRequest here. The exception handler here would be interesting in case we wanna have some kind of fallback (in the case of a fail when loading the validator)

Reply:
@italovalcy what do you think we should do in this case? Do not catch the exception (then the NApp will not load) or catch it and disable the validation?